superset/__init__.py

```diff
@@ -69,12 +69,21 @@ def get_css_manifest_files(filename):
     return entry_files.get('css', [])
 
 
+def filter_loaded_chunks(files, loaded_chunks):
```

Comment:
I find the name a little confusing, i.e., you’re returning the unloaded chunks, maybe something like `get_unloaded_chunks` makes more sense.